from exception import add 
a = (input("enter a:"))
b = (input("enter b:"))
c = add(a,b)
print(c)